src/Domain/Campaign.php

```diff
@@ -243,22 +260,43 @@ public function isReady(): bool
         return $this->ready;
     }
 
-    public function updateFromSfPull(
+    final public function updateFromSfPull(
```

Comment:
What's the reason for this change?

Reply:
Without this change psalm would show these errors:

```
ERROR: PropertyNotSetInConstructor
at /home/barney/projects/matchbot/src/Domain/Campaign.php:77:18
Property MatchBot\Domain\Campaign::$ready is not defined in constructor of MatchBot\Domain\Campaign or in any private or final methods called in the constructor (see https://psalm.dev/074)
    private bool $ready;


ERROR: PropertyNotSetInConstructor
at /home/barney/projects/matchbot/src/Domain/Campaign.php:80:18
Property MatchBot\Domain\Campaign::$isRegularGiving is not defined in constructor of MatchBot\Domain\Campaign or in any private or final methods called in the constructor (see https://psalm.dev/074)
    private bool $isRegularGiving;


ERROR: PropertyNotSetInConstructor
at /home/barney/projects/matchbot/src/Domain/Campaign.php:91:35
Property MatchBot\Domain\Campaign::$regularGivingCollectionEnd is not defined in constructor of MatchBot\Domain\Campaign or in any private or final methods called in the constructor (see https://psalm.dev/074)
    protected ?\DateTimeImmutable $regularGivingCollectionEnd;
```

Because if the method is not final then in principle a subclass could override it, which would mean those properties would never get set, violating our expectation that they will always be set when the object is constructed.